components/user-mgt/org.wso2.carbon.identity.user.pre.update.password.action/src/test/java/org/wso2/carbon/identity/user/pre/update/password/action/rule/PreUpdatePasswordActionRuleEvaluationDataProviderTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.user.pre.update.password.action.rule;
+
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.common.testng.WithCarbonHome;
+import org.wso2.carbon.identity.core.context.IdentityContext;
+import org.wso2.carbon.identity.core.context.model.Flow;
+import org.wso2.carbon.identity.rule.evaluation.exception.RuleEvaluationDataProviderException;
+import org.wso2.carbon.identity.rule.evaluation.model.Field;
+import org.wso2.carbon.identity.rule.evaluation.model.FieldValue;
+import org.wso2.carbon.identity.rule.evaluation.model.FlowContext;
+import org.wso2.carbon.identity.rule.evaluation.model.FlowType;
+import org.wso2.carbon.identity.rule.evaluation.model.RuleEvaluationContext;
+import org.wso2.carbon.identity.rule.evaluation.model.ValueType;
+import org.wso2.carbon.identity.user.pre.update.password.action.core.rule.PreUpdatePasswordActionRuleEvaluationDataProvider;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.Mockito.doReturn;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Test class for PreUpdatePasswordActionRuleEvaluationDataProvider.
+ */
+@WithCarbonHome
+public class PreUpdatePasswordActionRuleEvaluationDataProviderTest {
+
+    @Mock
+    private RuleEvaluationContext ruleEvaluationContext;
+    @Mock
+    private FlowContext flowContext;
+    @Mock
+    private Flow flow;
+
+    private PreUpdatePasswordActionRuleEvaluationDataProvider dataProvider;
+
+    @BeforeMethod
+    public void setUp() {
```

Comment:
looks like method not formatted

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6418/commits/867a9a807ddc7ab4dee5b0fdf8fc9a233b2cf669